feat : update readme cargo loading (#39)
eve-autonomy/v2i_interface: README.md

  - from user-defined broadcasting device
    - `v2i status` ([UDP](#v2i-status)) :<br>State from V2I infrastructure. It has an array structure to control the vehicle based on the state of multiple infrastructures.
- output
  - to [autoware.universe](https://github.com/autowarefoundation/autoware.universe/)
    - `/system/v2x/virtual_traffic_light_status` \[[tier4_v2x_msgs/msg/VirtualTrafficLightStateArray](https://github.com/tier4/tier4_autoware_msgs/blob/tier4/universe/tier4_v2x_msgs/msg/VirtualTrafficLightStateArray.msg)\]:<br>ROS2 interface from `v2i_status` (UDP).
  - to [cargo_loading_service](https://github.com/eve-autonomy/cargo_loading_service)
    - `/v2i/infrastructer_states` \[[v2i_interface_msgs/msg/InfrastructureStateArray.msg](https://github.com/eve-autonomy/v2i_interface_msgs/blob/main/msg/InfrastructureState.msg)\]:<br>ROS2 interface from `v2i_status` (UDP).
  - to user-defined broadcasting device
    - `v2i command` ([UDP](#v2i-command)) :<br>UDP protocol from `/awapi/tmp/infrastructure_commands`.

## Node Graph
### v2i interface ｍodule external specifications

> ![node graph](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/eve-autonomy/v2i_interface/main/docs/node_graph.pu)

Reviewer: ```suggestion
![node graph](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/eve-autonomy/v2i_interface/main/docs/node_graph_external_spec.pu)
```

Reviewer: I've confirmed that it's fixed in https://github.com/eve-autonomy/v2i_interface/pull/39/commits/ecc4a47ca0edcf036b816b130c64cbe0ac959e89 .

- input
  - from [autoware.universe](https://github.com/autowarefoundation/autoware.universe/)
    - `/awapi/tmp/infrastructure_commands` \[[tier4_v2x_msgs/msg/InfrastructureCommandArray](https://github.com/tier4/tier4_autoware_msgs/blob/tier4/universe/tier4_v2x_msgs/msg/InfrastructureCommandArray.msg)\]:<br>Control command to V2I infrastructure. It has an array structure to control multiple infrastructures at the same time.

>     - from [cargo_loading_service](https://github.com/eve-autonomy/cargo_loading_service)

Reviewer: ```suggestion
  - from [cargo_loading_service](https://github.com/eve-autonomy/cargo_loading_service)
```

    - `/autoware_state_machine/state` \[[autoware_state_machine_msgs/msg/StateMachine](https://github.com/eve-autonomy/autoware_state_machine_msgs/blob/main/msg/StateMachine.msg)\]:<br>State of the system.
  - from user-defined broadcasting device
    - `v2i status` ([UDP](#v2i-status)) :<br>State from V2I infrastructure. It has an array structure to control the vehicle based on the state of multiple infrastructures.
  - from [cargo_loading_service](https://github.com/eve-autonomy/cargo_loading_service)

>     - `/awapi/tmp/infrastructure_commands` \[[tier4_v2x_msgs/msg/InfrastructureCommandArray](https://github.com/tier4/tier4_autoware_msgs/blob/tier4/universe/tier4_v2x_msgs/msg/InfrastructureCommandArray.msg)\]:<br>Control command to V2I infrastructure. It has an array structure to control multiple infrastructures at the same time.

Reviewer: Inputs of the same system can be easily understood by the user by describing them consecutively.
Please move between L.13 and L.14.

Author: upda https://github.com/eve-autonomy/v2i_interface/pull/39/commits/92560c5be4b9910a50f968396e6d13beea742d9f

>   - from [cargo_loading_service](https://github.com/eve-autonomy/cargo_loading_service)
>     - `/awapi/tmp/infrastructure_commands` \[[tier4_v2x_msgs/msg/InfrastructureCommandArray](https://github.com/tier4/tier4_autoware_msgs/blob/tier4/universe/tier4_v2x_msgs/msg/InfrastructureCommandArray.msg)\]:<br>Control command to V2I infrastructure. It has an array structure to control multiple infrastructures at the same time.

Reviewer: The topic name is the same as autoware, but is it wrong?
```suggestion
  - from [cargo_loading_service](https://github.com/eve-autonomy/cargo_loading_service)
    - `/cargo_loding/infurastructre_commands` \[[tier4_v2x_msgs/msg/InfrastructureCommandArray](https://github.com/tier4/tier4_autoware_msgs/blob/tier4/universe/tier4_v2x_msgs/msg/InfrastructureCommandArray.msg)\]:<br>Control command to V2I infrastructure. It has an array structure to control multiple infrastructures at the same time.
```

> - output

Reviewer: It looks like there are more outputs for cargo_loading_service in the node graph, so please add a corresponding topic.

Author: update https://github.com/eve-autonomy/v2i_interface/commit/92560c5be4b9910a50f968396e6d13beea742d9f

Author: update https://github.com/eve-autonomy/v2i_interface/pull/39/commits/c120cc89c5b1e291a13483992f312f134a5fc7cf
